INSTALL.md

```diff
@@ -270,6 +271,33 @@ sudo docker run --rm -it --privileged \
   alpine:3.12
 ```
 
+## WSL - Binary
```

Comment:
Can you put `(Windows Subsystem for Linux)` here as well? Will make this easier to search for

Reply:
thx! PTAL.